tests/sdl2_key.c

```diff
@@ -28,6 +28,11 @@ int EventHandler(void *userdata, SDL_Event *event) {
         }
       }
       break;
+    case SDL_TEXTINPUT:
+      if (event->text.text[0] == 'A') {
+        printf("a\n");result *= 19;
+      }
```

Comment:
There's actually a bug that i'm sending keypress events for non-printable keys (arrows)